debian/vendor-rust.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -eu
+
+# Some crates are shipped with .a files, which get removed by the helpers during the package build as a safety measure.
+# This results in cargo failing to compile, since the files (which are listed in the checksums) are not there anymore.
+# For those crates, we need to replace their checksum with a more general one that only lists the crate checksum, instead of each file.
+CARGO_HOME=${HOME}/.cargo ${CARGO} vendor "${CARGO_VENDOR_DIR}"
+
+[ ! -e "${DH_CARGO_VENDORED_SOURCES}" ] || ${DH_CARGO_VENDORED_SOURCES}
+[ -e /usr/bin/jq ] || (echo "jq is required to run this script. Try installing it with 'sudo apt install jq'" && exit 1)
+
+for dep in $(ls vendor_rust -1); do
```

Comment:
Just a nit, but I think this could be just be:

```suggestion
for dep in vendor_rust/*; do
```